source/plugin/live.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Live
+group: Plugins
+order: 14
+---
+
+Use live plugin to animate cells and rows when they will be changed.
+
+
+{% docEditor "github/qgrid/ng2-example/tree/live-data-basic/latest" %}
+
+## Installation
+
+- For cell animation: Add `ng-template` inside `q-grid-column` with `change` input and add live-cell component inside your `ng-template`.
```

Comment:
Use `q-grid-live-cell` inside cell template to animate cell's content and `q-grid-live-row` for rows animation. 

should be enough